component-samples/demo/README.MD

```diff
@@ -262,7 +262,8 @@ The FDO PRI HTTP Java Device Sample currently supports `fdo_sys` module for inte
 
     *fetch* - This command returns the fetched file from client filesystem.
 
-    *owner_exec* - This command is a generic execution module that can perform various tasks, such as downloading files, running scripts, and executing commands
+    *owner_exec* - This command is a generic execution module that can perform various tasks, such as downloading files, running scripts, and executing commands. If you want to add some specific package edit the Dockerfile of the owner folder.
```

Comment:
can the last line modified as something like this "For executing specific package add the package in the respective Dockers."

Reply:
Done

Reply:
Done here - https://github.com/fido-device-onboard/pri-fidoiot/pull/657/commits/bec9ca826dab8877320cfa6b0cabb238c980ec52